buildozer/default.spec

```diff
@@ -263,7 +263,10 @@ ios.kivy_ios_branch = master
 #ios.ios_deploy_dir = ../ios_deploy
 # Or specify URL and branch
 ios.ios_deploy_url = https://github.com/phonegap/ios-deploy
-ios.ios_deploy_branch = 1.7.0
+ios.ios_deploy_branch = 1.10.0
+
+# (bool) Whether or not to sign the code
+ios.codesign.allowed = false
```

Comment:
Two things here. First, I'm ditching the old C-style `0` or `1` fashion and I plan to do so through the entire code base in the future. The good thing is it's retro compatible, so people it won't change in behavior for people having `0` or `1` in their settings https://docs.python.org/3/library/configparser.html#supported-datatypes
Second I set this up to `false` by default while introducing the setting. This is a change in behavior, but will lead to less complains/questions from our users. This is because most of the time people just want to try things locally and don't need to sign and publish to the Apple store. As for the ones who went through all the struggle and actually made it to be 99% ready to publish to the store, they're usually good enough to figure out they need to change this setting to sign their app

Reply:
I honestly though that codesign would be used in release only, never in debug mode